### DISCUSSION TITLE
Cursor: grab  Can we change it to different cursor ?

### DISCUSSION THREAD

**Discussion by TalRodin** [google/model-viewer#2911] [Q&A]
Hi, the question in the title. Thank you for your response.

**Comment by elalish** [comment]
No, that's not exposed. What  do you want to change it to?

**Reply by cw789** [reply]
Is the cursor still not exposed? I think so, didn't find anything.
I try to change the grab and grabbing cursor with custom SVG icons.

`cursor: url("grab.svg") 40 40, grabbing`

**Reply by elalish** [reply]
Indeed it is still not exposed. PRs welcome.

**Reply by cw789** [reply]
Thanks. I'll see what I can do when polish phase of project is coming.

Edit: Time is going to be to short to tackle this.
